api/src/main/java/marquez/cli/DbMigrateCommand.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018-2022 contributors to the Marquez project
```

Comment:
Minor: _Copyright 2018-2024_...

Reply:
Actually, we should probably check other files that have an older copyright. Let's do this as follow up instead.